.travis.yml

```diff
@@ -165,15 +165,21 @@ scripts:
 
 matrix:
   include:
+    - <<: *opam
+      os: linux
+      env: COMPILER=4.12.0+trunk REPOSITORIES=--repo=default,beta=git://github.com/ocaml/ocaml-beta-repository.git
+    - <<: *opam
+      os: linux
+      env: COMPILER=4.11.1
     - <<: *opam
       os: osx
-      env: COMPILER=4.10.0 LIBEV=no
+      env: COMPILER=4.10.1 LIBEV=no
     - <<: *opam
       os: linux
-      env: COMPILER=4.10.0 LIBEV=no PPX_LET=yes
+      env: COMPILER=4.10.1 LIBEV=no PPX_LET=yes
     - <<: *opam
       os: linux
-      env: COMPILER=4.09.0 DOCS=yes
+      env: COMPILER=4.09.1 DOCS=yes
```

Comment:
Can you tweak the rows in `allow_failures`, so that it recognizes the 4.09 row again? We should also add 4.10 and 4.12 to `allow_failures`. Travis is set up to require on Linux the least and greatest OCaml release that we test (4.02.3 and 4.11.1, respectively, in this PR), and the rest are in `allow_failures` so that we can get build feedback faster if Travis has a build backlog.